test/latency/latency.json

```diff
@@ -0,0 +1,40 @@
+{
+    "docker-config": {
+        "request-timeout": 10000000000,
+        "docker-client-version": "1.24",
+        "privileged": true,
+        "map-volumes": [
+            "/sys/bus/pci/drivers:/sys/bus/pci/drivers",
+            "/sys/kernel/mm/hugepages:/sys/kernel/mm/hugepages",
+            "/sys/devices/system/node:/sys/devices/system/node",
+            "/dev:/dev"
+        ],
+        "docker-port": 2375,
+        "pktgen-port": 22022
+    },
+    "tests": [
+        {
+            "name": "latency-test",
+            "test-time": 60000000000,
+            "test-type": "TEST_TYPE_SCENARIO",
+            "test-apps": [
+                {
+                    "host-name": "nonpcmp2",

```

Comment:
Let's use generic hostname1 and hostname2 in test configs because that is what we have in documentation at the moment.